### ISSUE TITLE
Move change log to github and automate

### ISSUE THREAD

**Issue by ramonawalls** [GenomicsStandardsConsortium/mixs#343]
It is currently at [https://docs.google.com/spreadsheets/d/1G5ri-vUERgy3zfzj_MQsZEff0K8mT1LT/edit#gid=1075103823](https://www.google.com/url?q=https://docs.google.com/spreadsheets/d/1G5ri-vUERgy3zfzj_MQsZEff0K8mT1LT/edit%23gid%3D1075103823&sa=D&source=docs&ust=1645288983809267&usg=AOvVaw2jreTKxTteWLYL4jQbgwt0)

We can copy this file to the repo for now, but we need to automate the change log going forward to make sure it captures all changes. 

Todo:
- [ ] move file to github
- [ ] devise plan for automation
- [ ] implement plan

**Comment by ramonawalls** [comment]
change log added with commit [025e5ce](https://github.com/GenomicsStandardsConsortium/mixs/commit/025e5ce48ba9a48f0f9bd6eeb43d22c464eb9ca2)

**Comment by ramonawalls** [comment]
Our current plan is to use github issues, branches, PRs, and releases to autogenerate change logs. Documentation in progress at https://github.com/GenomicsStandardsConsortium/mixs/blob/606-mixs-editing-workflow-and-policy/src/docs/edit_workflow.md. 

Should close this issue when #606 is closed.

**Comment by ramonawalls** [comment]
Should be documented in [PR](https://github.com/GenomicsStandardsConsortium/mixs/tree/606-mixs-editing-workflow-and-policy)

**Comment by turbomam** [comment]
I don't believe the changelog referenced in the original post is relevant any more
